admin/database/database.go

```diff
@@ -649,21 +650,34 @@ type InsertProjectInviteOptions struct {
 type Bookmark struct {
 	ID            string
 	DisplayName   string    `db:"display_name"`
+	Description   string    `db:"description"`
 	Data          []byte    `db:"data"`
 	DashboardName string    `db:"dashboard_name"`
 	ProjectID     string    `db:"project_id"`
 	UserID        string    `db:"user_id"`
+	IsGlobal      bool      `db:"is_global"`
```

Comment:
Would suggest `Shared` instead of `IsGlobal` since "global" implies everybody has access to it, whereas it's actually just the people with access to the resource who gets access to it